docs/kubernetes-versions.md

```diff
@@ -1,71 +1,452 @@
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes-test-linux-ppc64le.tar.gz.sha512
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes-test-linux-s390x.tar.gz
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes-test-linux-s390x.tar.gz.sha256
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes-test-linux-s390x.tar.gz.sha512
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes-test-portable.tar.gz
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes-test-portable.tar.gz.sha256
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes-test-portable.tar.gz.sha512
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes-test-windows-amd64.tar.gz
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes-test-windows-amd64.tar.gz.sha256
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes-test-windows-amd64.tar.gz.sha512
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes.tar.gz
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes.tar.gz.sha256
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/kubernetes.tar.gz.sha512
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/bin/
+gs://kubernetes-release-dev/ci/v1.20.0-alpha.0.391+575c4925be8c39/extra/
 ```
+
+</details>
+
+A general, non-exhaustive list of the expected artifacts of a Kubernetes build
+can be found [here](https://git.k8s.io/sig-release/release-engineering/artifacts.md).
+
+## Marker types
+
+Version markers are broken into two primary types: `CI` and `release` markers
+
+### CI
+
+**GCS Bucket:** `kubernetes-release-dev`
+
+**Directory:** `ci`
+
+#### latest
+
+**Path:** `https://storage.googleapis.com/kubernetes-release-dev/ci/latest[-x.y].txt`
+
+**kubekins-e2e `extract`:** `ci/latest[-x.y].txt`
+
+`latest` markers reference cross builds generated via the `ci-kubernetes-build`
+jobs, which run approximately every hour.
+
+This version marker exists for all active Kubernetes release branches.
+
+#### latest-fast
+
+**Path:** `https://storage.googleapis.com/kubernetes-release-dev/ci/latest-fast.txt`
+
+**kubekins-e2e `extract`:** `ci/latest-fast.txt`
+
+`latest-fast` markers reference linux/amd64-only builds generated via the
+`ci-kubernetes-build-fast` job, which run approximately every 5 minutes.
+
+This version marker only exists for the `master` branch.
+
+#### **DEPRECATED** - generic
+
+**Path:** `https://storage.googleapis.com/kubernetes-release-dev/ci/k8s-<generic-version>.txt`
+
+**kubekins-e2e `extract`:** `ci/k8s-<generic-version>.txt`
+
+The following generic markers are available:
+
+- `k8s-master`
+- `k8s-beta`
+- `k8s-stable1`
+- `k8s-stable2`
+- `k8s-stable3`
```

Comment:
I think @detiber 's suggestion is a bit more intuitive:
```suggestion
- `k8s-stable`
- `k8s-stable-1`
- `k8s-stable-2`
...
```

Reply:
This documents the current state, where "current" is what exists today in addition to what would happen once this PR merges.

There is no `k8s-stable` marker.

Reply:
Got it. Thanks.